packages/react/src/components/field/useField.ts

```diff
@@ -1,8 +1,18 @@
 import { FieldState } from '@onfido/castor';
 import { createContext, useContext } from 'react';
+import { useForm } from '../form/useForm';
 
 const FieldContext = createContext({} as FieldState);
 
-export const useField = () => useContext(FieldContext);
+export const useField = () => {
+  const form = useForm();
+  const field = useContext(FieldContext);
+
+  return { ...omitNullish(form), ...omitNullish(field) };
+};
 
 export const FieldProvider = FieldContext.Provider;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const omitNullish = (obj: Record<string, any>) =>
```

Comment:
Odd naming, what's wrong with `omitNull`?

Reply:
It also omits `undefined` 🤷🏻 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator